xray/plotting/plotting.py

```diff
@@ -61,10 +61,10 @@ def plot(darray, ax=None, rtol=0.01, **kwargs):
     =============== =========== ===========================
```

Comment:
Maybe better to rename this `plot/plot` to be consistent with `DataArray.plot`. Thoughts?